main.js

```diff
@@ -0,0 +1,54 @@
+// DOM utility functions
+const addClass = (el, newClass) => {
+  el.className += ` ${newClass}`;
+};
+
+const removeClass = (el, oldClass) => {
```

Comment:
Right yeah. Essentially, `addClass(el, newClass)` and the other are now bound for good. As in, no one can now reassign those names to different functions. The internals of `const` variables can be changed and manipulated (it's not a primitive - numbers, strings, etc.), but you can't rebind it as a whole.

Because they're scoped globally here in this example, later in other functions or blocks, you can be fully certain that these functions are going to stay put.

Here's also an article that argues that `const` should be the default you use, until you find that you need to reassign the value later - then use `let`: https://mathiasbynens.be/notes/es6-const